src/javaExercises/stairCase/StairCase.java

```diff
@@ -0,0 +1,22 @@
+package javaExercises.stairCase;
+
+import java.util.Scanner;
+
+public class StairCase {
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input desired staircase height:");
+        int n = sc.nextInt();
```

Comment:
give more meaningful names to your variables

Reply:
Done